### DISCUSSION TITLE
Two column layout

### DISCUSSION THREAD

**Discussion by davosian** [Azure/api-management-developer-portal#1692] [General]
I would like to create a layout which contains the navigation vertically on the left hand side and the right hand side should be the content area. Is this possible? 

When I tried, I was only able to add additional widgets above or below the layout's content area, but there does not seem to be a way to move the content area into another section.

**Comment by azaslonov** [comment]
Hi @davosian, it is possible. You can add a section with the required layout and drag'n'drop the Content placeholder to required area. However, it seems like we have a bug and that Content is not draggable like it used to be. We're planning to fix it this week and ship with the next release, so I would expect it to be out in all regions by 4/1.
